pom.xml

```diff
@@ -186,12 +181,6 @@
             <groupId>commons-validator</groupId>
             <artifactId>commons-validator</artifactId>
             <version>1.6</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>commons-collections</groupId>
-                    <artifactId>commons-collections</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
I was wondering if perhaps one reason for excluding commons-validator’s version of commons-collections might have been that it was on an older version with security vulnerabilities but it seems commons-validator 1.6 depends on the same 3.2.2 version of commons-collections as we were already using, so nothing changes there.